File: PSP/SpringApiRest/src/main/java/org/example/springdemo/data/entity/EjercicioEntity.java
package org.example.springdemo.data.entity;


import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class EjercicioEntity {
    private int id;
    private int entrenamientoId;
    private String nombre;
    private int repeticiones;
    private int series;


    public EjercicioEntity() {}

}

File: MOVILES/Navigation/app/src/main/java/com/example/navigation/data/local/entities/EjercicioEntity.kt
package com.example.navigation.data.local.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(
    tableName = "ejercicios"
)
data class EjercicioEntity(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val nombre: String,
    val grupoMuscular: String? = null,
    val descripcion: String? = null,
    val iconoResName: String? = null
)
